src/utils/tasks/taskSkillsTree.goerli.ts

```diff
@@ -2,9 +2,7 @@
  * Skills Tree For the Goerli Network
  */
 
-import { ConsumableItem } from '~core/ItemsList';
-
-const taskSkills: ConsumableItem[] = [
+const taskSkills = [
```

Comment:
Are these task skills still used anywhere or will be used in the future? I did a simple variable search of `taskSkill` & `getTaskSkillsTreeInUse` and didn't see any except the declaration ones. But maybe I am missing something?

Reply:
Not for now, but in task requirements was written to keep skills for future :)

Reply:
We won't. When I first wrote that I had other plans :)

Those skills are generated from the contracts, so they don't really matter.